RoadMap_SwiftUI/RoadMap_SwiftUI/BottomSheetView.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  BottomSheetView.swift
+//  RoadMap_SwiftUI
+//
+//  Created by Артур Щукин on 31.01.2023.
+//
+
+import SwiftUI
+
+struct BottomSheetView<Content: View>: View {
+    let content: () -> Content
+    
+    @GestureState private var gestureOffset = CGSize.zero
+    @State private var currentMenuOffsetY: CGFloat = 0.0
+    @State private var lastMenuOffsetY: CGFloat = 0.0
+    
+    
+    var dragGesture: some Gesture {
+        DragGesture()
+            .updating($gestureOffset) { value, state, transaction in
+                state = value.translation
+                onChangeMenuOffset()
+            }
+            .onEnded { value in
+                let maxHeight = UIScreen.main.bounds.height / 6
+                withAnimation {
+                    if -currentMenuOffsetY > maxHeight / 2 {
+                        currentMenuOffsetY = -maxHeight
+                    } else {
+                        currentMenuOffsetY = 0
+                    }
+                    lastMenuOffsetY = currentMenuOffsetY
+                }
+            }
+    }
+    
+    func onChangeMenuOffset() {
+        DispatchQueue.main.async {
+            self.currentMenuOffsetY = gestureOffset.height + lastMenuOffsetY
+        }
+    }
+    
+    var body: some View {
```

Comment:
```suggestion
    var body: some View {
```
Боди наверх, и марки добавь